### ISSUE TITLE
invalid literal for int() with base 10:

### ISSUE THREAD

**Issue by thomasyu888** [genome-nexus/annotation-tools#7]
```
Traceback (most recent call last):
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1430, in <module>
    main()
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1427, in main
    generate_maf_from_input_data(input_directory, output_directory, extensions_list, center_name, sequence_source)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1393, in generate_maf_from_input_data
    maf_data = extract_vcf_data_from_file(os.path.join(input_directory, filename), center_name, sequence_source)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1327, in extract_vcf_data_from_file
    maf_record = create_maf_record_from_vcf(sample_id, center_name, sequence_source, vcf_data, is_germline_data, matched_normal_sample_id, tumor_sample_data_col)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1277, in create_maf_record_from_vcf
    resolve_vcf_counts_data(vcf_data, maf_data, matched_normal_sample_id, tumor_sample_data_col)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1178, in resolve_vcf_counts_data
    (t_ref_count, t_alt_count, t_depth) = resolve_vcf_allele_depth_values(tumor_sample_format_data, vcf_alleles, variant_allele_idx, vcf_data)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1117, in resolve_vcf_allele_depth_values
    allele_depth_values[variant_allele_idx] = "%.0f" % (int(mapped_sample_format_data["FA"]) * int(mapped_sample_format_data["DP"]))
ValueError: invalid literal for int() with base 10: '0.049'
```

I am gathering examples from all the errors I see, but thought I would record the error

**Comment by thomasyu888** [comment]
```
Traceback (most recent call last):
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1430, in <module>
    main()
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1427, in main
    generate_maf_from_input_data(input_directory, output_directory, extensions_list, center_name, sequence_source)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1393, in generate_maf_from_input_data
    maf_data = extract_vcf_data_from_file(os.path.join(input_directory, filename), center_name, sequence_source)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1327, in extract_vcf_data_from_file
    maf_record = create_maf_record_from_vcf(sample_id, center_name, sequence_source, vcf_data, is_germline_data, matched_normal_sample_id, tumor_sample_data_col)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1277, in create_maf_record_from_vcf
    resolve_vcf_counts_data(vcf_data, maf_data, matched_normal_sample_id, tumor_sample_data_col)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1178, in resolve_vcf_counts_data
    (t_ref_count, t_alt_count, t_depth) = resolve_vcf_allele_depth_values(tumor_sample_format_data, vcf_alleles, variant_allele_idx, vcf_data)
  File "genie-annotation-pkg/standardize_mutation_data.py", line 1138, in resolve_vcf_allele_depth_values
    ref_count != "" and int(ref_count) > int(vcf_dp_value)
ValueError: invalid literal for int() with base 10: '.'
```

**Comment by ao508** [comment]
resolved by pr #9
